packages/ui-library/src/components/feedback/loader/index.ts

```diff
@@ -4,9 +4,8 @@ import { customElement, property } from 'lit/decorators.js';
 import { styleCustomLight, styleCustomDark } from './index.css';
 import { ActionSizesType, FeedbackVariantType } from '../../../globals/types';
 import { ThemeType } from '../../../foundation/_tokens-generated/index.themes';
-import { genericBlrComponentRenderer } from '../../../utils/typesafe-generic-component-renderer';
 
-const TAG_NAME = 'blr-loader';
+export const TAG_NAME = 'blr-loader';
```

Comment:
maybe we should rename these because now we have multiple `TAG_NAME`s in our global namespace

Reply:
we should rename it: yes.

does it collide or pollute? no :) 

the index.ts defines whats exported to build, and its not imported/exported there.
also the build is setup up to not use global name space.

Reply:
double checked: no TAG_NAME inside the build at all :)

Reply:
To be honest I didn't have the build in mind, I just meant that if you try to import `TAG_NAME` you now the `ts-server` suggestions are confusing

Reply:
yes, i completely agree on that point. i also dont want to know what happens if you move component folders around. they should be unique. but maybe not on this branch wich is already hard to rebase on